app/src/main/res/drawable/ic_undo.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="15dp" android:viewportHeight="512"
```

Comment:
where did you get this file from?

Reply:
Is this a common icon or a third party image?

Reply:
This one is from www.flaticon.com, a free icon.

Reply:
Let's not use any resources from third party unless we are absolutely sure it is needed. The reason is that they might be protected by copyright and not be allowed to be used by software companies. 

If you need an icon, tell us and we can decide what to do with it together.

Reply:
Ok, will use icons from https://material.io/resources/icons/?style=baseline

Reply:
Makes sense